content/en/docs/concepts/security/mutating-webhooks-good-practices.md

```diff
@@ -0,0 +1,585 @@
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><a href="/docs/reference/access-authn-authz/extensible-admission-controllers/">Mutating admission webhook</a></td>
+      <td>Intercept API requests before admission and modify as needed using
+        custom logic.</td>
+      <td><ul>
+        <li>Make critical modifications that must happen before resource
+          admission.</li>
+        <li>Make complex modifications that require advanced logic, like calling
+          external APIs.</li>
+      </ul></td>
+    </tr>
+    <tr>
+      <td><a href="/docs/reference/access-authn-authz/mutating-admission-policy/">Mutating admission policy</a></td>
+      <td>Intercept API requests before admission and modify as needed using
+        Common Expression Language (CEL) expressions.</td>
+      <td><ul>
+        <li>Make critical modifications that must happen before resource
+          admission.</li>
+        <li>Make simple modifications, such as adjusting labels or replica
+        counts.</li>
+      </ul></td>
+    </tr>
+    <tr>
+      <td><a href="/docs/reference/access-authn-authz/extensible-admission-controllers/">Validating admission webhook</a></td>
+      <td>Intercept API requests before admission and validate against complex
+        policy declarations.</td>
+      <td><ul>
+        <li>Validate critical configurations before resource admission.</li>
+        <li>Enforce complex policy logic before admission.</li>
+      </ul></td>
+    </tr>
+    <tr>
+      <td><a href="/docs/reference/access-authn-authz/validating-admission-policy/">Validating admission policy</a></td>
+      <td>Intercept API requests before admission and validate against CEL
+        expressions.</td>
+      <td><ul>
+        <li>Validate critical configurations before resource admission.</li>
+        <li>Enforce policy logic using CEL expressions.</li>
+      </ul></td>
+    </tr>
+  </tbody>
+</table>
+
+In general, use _webhook_ admission control when you want an extensible way to
+declare or configure the logic. Use built-in CEL-based admission control when
+you want to declare simpler logic without the overhead of running a webhook
+server. The Kubernetes project recommends that you use CEL-based admission
+control when possible.
+
+### Use built-in validation and defaulting for CustomResourceDefinitions {#no-crd-validation-defaulting}
+
+If you use
+{{< glossary_tooltip text="CustomResourceDefinitions" term_id="customresourcedefinition" >}},
+don't use mutating webhooks to validate values in CustomResource specifications
+or to set default values for fields. Kubernetes lets you define validation rules
+and default field values when you create CustomResourceDefinitions.
+
+To learn more, see the following resources:
+
+* [Validation rules](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules)
+* [Defaulting](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting)
```

Comment:
If we can link to a concept page here, even better (there may not be a good concept page yet).

Reply:
We don't. It looks like the "Advanced topics" main section in the task page is mostly conceptual content, so probably a good candidate for moving to a concept page. We have https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/ but that only lightly touches on CRD design